appservice/src/createAppService/AppServicePlanCreateStep.ts

```diff
@@ -21,7 +21,7 @@ export class AppServicePlanCreateStep extends AzureWizardExecuteStep<IAppService
 
     public async execute(context: IAppServiceWizardContext, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
         const newPlanName: string = nonNullProp(context, 'newPlanName');
-        const rgName: string = nonNullValueAndProp(context.resourceGroup, 'name');
```

Comment:
I removed `../utils/nonNull`; this `nonNullValueAndProp` function was called only here and was defined as `nonNullProp(nonNullValue(...))`. Rather than have a whole file and function for one call...I just put the one-liner here.